app/src/views/OperationalLearning/Filters/index.tsx

```diff
@@ -116,63 +106,109 @@ function Filters(props: Props) {
         );
     }, [onChange]);
 
+    const handleStartDateSelect = useCallback((
+        newValue: string | undefined,
+        key: 'appealStartDateAfter',
+    ) => {
+        onChange(
+            newValue,
+            key,
+            newValue,
+        );
+    }, [onChange]);
+
+    const handleEndDateSelect = useCallback((
+        newValue: string | undefined,
+        key: 'appealStartDateBefore',
+    ) => {
+        onChange(
+            newValue,
+            key,
+            newValue,
+        );
+    }, [onChange]);
+
+    const handleSearch = useCallback((
+        newValue: string | undefined,
+        key: 'appealSearchText',
+    ) => {
+        onChange(
+            newValue,
+            key,
+        );
+    }, [onChange]);
+
```

Comment:
Why not directly use `onChange` on the components?

Reply:
@tnagorra we need to show the labels of the selection as pills component.

Reply:
The function signature for `handleSearch` and `onChange` are the same so we should be able to just use `onChange` instead

Reply:
@tnagorra here a typing issue arises.